bfdd/bfdd_nb_config.c

```diff
@@ -386,18 +538,24 @@ int bfdd_bfd_profile_required_receive_interval_modify(
 	struct nb_cb_modify_args *args)
 {
 	struct bfd_profile *bp;
+	uint32_t min_rx;
 
 	switch (args->event) {
 	case NB_EV_VALIDATE:
+		min_rx = yang_dnode_get_uint32(args->dnode, NULL);
 		break;
 
 	case NB_EV_PREPARE:
 		/* NOTHING */
 		break;
 
 	case NB_EV_APPLY:
+		min_rx = yang_dnode_get_uint32(args->dnode, NULL);
 		bp = nb_running_get_entry(args->dnode, NULL, true);
-		bp->min_rx = yang_dnode_get_uint32(args->dnode, NULL);
+		if (bp->min_rx == min_rx)
+			return NB_OK;
+
+		bp->min_rx = min_rx;
 		bfd_profile_update(bp);
```

Comment:
ditto: separate commit

Reply:
you seem to have removed those changes.
if the behavior can be kept unchanged, it is ok to me.